app/styles/messaging-styles.css

```diff
@@ -10,6 +10,18 @@
 .conversation-height {
   height: calc(100vh - 65px);
 }
+
+.notification-dot {
+  position: absolute;
+  height: 1rem;
+  width: 1rem;
+  border-radius: 50%;
+  border-width: 2px;
+  border-color: #2f855a;
+  background-color: #54bb89;
+  box-shadow: 0 0 1px black;
+  z-index: 50;
+}
```

Comment:
Now that we have a component this seems out of place 😅 

But you did some creative thinking with the new props so why not declare this is the `NotificationDot.tsx` file and interpolate the passed prop values straight into it?

Probably remove the option for sizing though - I think the designers will want to keep that uniform, but we might well need to adjust the position when we place it in different areas so keep that one :)